database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -13,7 +13,8 @@ public function up(): void
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('name_first');
```

Comment:
We don't want to modify old migrations. We need a new migration that drops the name column, and adds the new columns.